release-tool

```diff
@@ -302,51 +324,45 @@ checkSnapcraft() {
 }
 
 checkTransifexCommandExists() {
-    command -v tx > /dev/null
-    if [ 0 -ne $? ]; then
+    if ! which tx > /dev/null; then
```

Comment:
I don't think our release-tool is POSIX-compatible anyway, which is why we use bash as interpreter, not sh. Also redirecting STDERR to STDOUT for these checks looks a bit over-pedantic to me.

Reply:
I'll accept it, but if it breaks on some niche Linux distro I'm directing the complaints to you :wink:

Reply:
I'd doubt that happens. which is just shorter and we are only interested in its return value, not its output. But I replaced it with a custom cmdExists function now, which wraps command -v. However, for the test for linuxdeploy/linuxdeploy-plugin-qt, I *have* to use which, because command is only a shell built-in, whereas which is an actual program and using command would fail with docker run, unless I wrap it into bash -c "...".